framework/encode/dx12_state_tracker.h

```diff
@@ -250,6 +256,7 @@ class Dx12StateTracker
     std::atomic_uint64_t accel_struct_id_;
 };
 
+bool IsResourceUsedForAccelerationStructure(format::HandleId id);
```

Comment:
Can this function signature be removed and then the implementation moved from the bottom of dx12_state_tracker.cpp to the top, just after the `#include`s and `GFXRECON_BEGIN_NAMESPACE` calls? That way this helper function stays local to Dx12StateTracker where it is used.

Reply:
Done.